### DISCUSSION TITLE
Strongswan with radius that authenticates agaisnt AD

### DISCUSSION THREAD

**Discussion by Nelse74** [strongswan/strongswan#1033] [Q&A] [answered]
Good morning,

I'm new in the VPN and Network world. I was tasked to use a VPN for remote access for Windows clients.
The users will use a Windows 10 laptop and IOS mobile phone, and I would like to use Strongswan with EAP-PEAP and a radius that authenticate against AD. The mean is for users to login with their Windows credential (during login session) and that it automatically connect them to their domain session. I've heard of xauth but it uses ikev1, and eap-gtc isn't supported on Windows (as I heard).
So I intend to use the eap-radius plugin for the strongswan to retrieve the user domain credential that radius got by communicating with the AD.
I wanted to know if I'm going the right way, and if it was possible.

Thanks for any tips, leads and answer. Have a nice day !

**Comment by tobiasbrunner** [comment, marked as answer]
Not sure about using EAP-PEAP (don't know if iOS supports this, maybe via custom profiles). But using EAP-MSCHAPv2 or EAP-TLS should be no problem (also see [the docs for the plugin](https://docs.strongswan.org/docs/5.9/plugins/eap-radius.html)).

**Reply by Nelse74** [reply]
Hello,
thanks for the answer !
